app/furniture/journal/entry_component.rb

```diff
@@ -0,0 +1,30 @@
+class Journal
+  class EntryComponent < ApplicationComponent
+    include RendersMarkdown
+    attr_accessor :entry
+
+    def initialize(*args, entry:, **kwargs)
+      self.entry = entry
+      super(*args, **kwargs)
+    end
+
+    def body_html
```

Comment:
What about `content_html` not to confuse with the`<body>` tag. This might be more consistent with other components that define header/content/footer slots?

Reply:
Ah, I like that! I will do a follow up PR to rename `Entry#body` to `Entry#content`